app/controllers/conference_controller.rb

```diff
@@ -21,10 +25,40 @@ def schedule
     conf_end = 20
     @conf_period = conf_end - @conf_start
 
-    if @dates == Date.current
-      @today = Date.current.strftime('%Y-%m-%d')
+    @day = Time.find_zone(@conference.timezone).today
+    # the schedule takes you to today if it is a date of the schedule
+    if @dates.include? @day
+      @has_date = true
+      current_hour = DateTime.now.in_time_zone(@conference.timezone).strftime('%H').to_i
```

Comment:
But I think that when you are in a conference you want to see what is taking place now or what is going to take place and you usually use your mobile phone for that. That means that if it is 20:00 you will take the user to 9:00 and he would have to click next 11 times. I really think that it is useful.